### ISSUE TITLE
Doesn't work on Sponge server

### ISSUE THREAD

**Issue by PixelmonPRO** [Shoxie12/MCDJ#4]
Good day! On a server 1.12.2 with the `Sponge`, when trying to interact with `Music Generator`, nothing happens and an error is thrown: https://gist.github.com/PixelmonPRO/9d53db5acc0a34a1fc09d82fb8e00ced

Java: `1.8.0_232`
Minecraft Version: `Minecraft 1.12.2`
Sponge Version: `SpongeForge 1.12.2-2838-7.3.0`

**Comment by Shoxie12** [comment]
This issue is related to displaying a message to the player about an empty playlist. Now you will see this message every time you try to interact with _Music Generator_ when no music has been added to the game.

**Comment by PixelmonPRO** [comment]
> This issue is related to displaying a message to the player about an empty playlist. Now you will see this message every time you try to interact with _Music Generator_ when no music has been added to the game.

I need to add music from the server or client side?
From the client's side, I added 1 music track, there were no problems in the single player game :)

UPD: I figured it out. Music needs to be loaded from the server and client side. If only from the server side, at login it gives an error 
UPD 2: Found a bug that when trying to write a disc, 0 track is always recorded: https://www.youtube.com/watch?v=217jiCza6xQ
